src/decoders/crx.cpp

```diff
@@ -2801,6 +2803,15 @@ int LibRaw::crxParseImageHeader(uchar *cmp1TagData, int nTrack)
   hdr->hasTileCols = cmp1TagData[27] >> 7;
   hdr->hasTileRows = (cmp1TagData[27] >> 6) & 1;
   hdr->mdatHdrSize = sgetn(4, cmp1TagData + 28);
+  int extHeader = cmp1TagData[32] >> 7;

```

Comment:
@markreidvfx Can you please provide an update for https://github.com/lclevy/canon_cr3 CMP1 specification for these new fields? Would be very helpful as I don't find some source for this.

Reply:
Yes I figured that stuff out on my own so it's probably not documented anywhere. I'll take a look, there might be a couple other things I can add. Like I've also found the colorspace, colormatrix and canonLog bytes in the CMT3 atom, but oddly none of the tif tags point to there location, still trying to solve that mystery.

Reply:
Did you try this tool to parse CRM ?https://github.com/lclevy/canon_cr3/blob/master/parse_cr3.py
 Or exiftool?

Reply:
@lclevy yes I have. parse_cr3.py has a bunch of exceptions when parsing the CRM files, but I haven't spent the time to debug.   exiftool parses the file, but not the data I'm looking for it to find.

Reply:
well, looks like I was just interpreting the tiff tag size wrong, tag 0x4026 in the CMT3 atom seems to have the colorspace info I'm looking for.

Reply:
So, do we need to parse MediaType==3 for makernotes, or this code may be removed?

Reply:
Sorry, please ignore the previous message:  parsing code is just moved from parseCR3 to selectCRXtrack, my question is wrong

Reply:
No I think its needed. Each frame has different makenote samples, things like focal length, focal distance can change. Personally that is data I use for my applications.

Reply:
What kind of application?

Reply:
Knowing the per frame focal length is useful for 3d tracking.